en/micro-integrator/docs/develop/create-integration-project.md

```diff
@@ -0,0 +1,117 @@
+
+An integration project is the recommended way of creating an “Integration Solution” for the WSO2 Enterprise Integrator as it simplifies the CICD workflow.
+
+**To create an integration project:**
+
+1. [Download](https://wso2.com/integration/integration-studio/) and [install WSO2 Integration Studio](installing-WSO2-Integration-Studio/).
+
+2. Open WSO2 Integration Studio and click **New Integration Project** in the **Getting Started** view as shown below.
+    <img src="../../assets/img/new-project/new-integration-project.png" title="New Integration Project" width="700" alt="New Integration Project"/>
+
+3. In the **New Integration Project** dialog that opens, enter a name for your integration project. Select the relevant check boxes if you want to create **Registry Resources**, **Connector Exporter**, **Docker Exporter**, or **Kubernetes Exporter** in addition to the **ESB Configs** and **Composite Exporter**.
+    <img src="../../assets/img/new-project/create-a-project.png" title="Create a New Integration Project" width="700" alt="Create a New Integration Project"/>
+
+## Sub projects
+
+An integration project can consist of multiple sub-projects. So multiple small projects can exist under a single integration project, where each of these can be dependent on each other and can be grouped together. However, it is not necessary that all sub-projects in an integration project be dependent on every other sub-project.
+
+To add sub-projects to an existing integration project, right click on the integration project and hover over **New** to see the available project creation options.
+
+<img src="../../assets/img/new-project/new-sub-project.png" title="Add a New Sub Project" width="250" alt="Add a New Sub Project"/>
+
+Once you create the new sub project, you can see this nested under your integration project folder in the project explorer.
+
+## About Maven Multi Module integration projects
+
+The Maven Multi Module (MMM) integration project is the parent project in an integration solution and sub-projects can be added under this parent project. By default, an integration project is an MMM project unless you specify otherwise. 
+
+By building the parent MMM project, you can build all the sub-projects in the integration solution simultaneously. This allows you to seamlessly push your integration solutions to a CI/CD pipeline. Therefore, it is recommended as a best practice to create your Config project and other projects inside an MMM integration project.
+
+This allows you to manage multiple projects such as Config projects, Composite Application projects, and Registry Resource projects as a single entity.
+
+Although the recommended approach is to create an integration project, which essentially has all the functionality of an MMM project, you can also create an MMM project separately.
+
+**To create the MMM project**:
```

Comment:
IMO, this needs to be subsection under 'Creating the Integration Project'.

"The Integration project is a MMM project. Using the the 'New Integration Project' option, you can create the complete project including all the required modules for your integration solution.
Alternatively, you can create an individual MMM project and then individually add the required modules."

wdyt?

Reply:
Will think about it and see if it can be merged into a single section.

Reply:
@nilminiwso2 shall we keep this as a separate section? Because this is conceptually not the same as an integration project even if it has the same functionality and can be alternate flow in the same story. Having it as a section of its own illustrates that difference.

IMO the ideal solution is to remove this completely from the UI but that's not an option as we need to have this for continuity purposes for existing users.